### PR TITLE
[#7592] Add github action for Ubuntu 24.04 (main)

### PR DESCRIPTION
**Pull request by SwooshyCueb**
In service of #7592

### CODE REVIEW

**Comment by SwooshyCueb** [comment]
Ready for review

**Review by alanking — COMMENTED** [review]
Agreed on spacing comment. Otherwise looks good

File: .github/workflows/build-irods-ubuntu.yml [hunk 1]
```diff
@@ -7,21 +7,15 @@ jobs:
     container: ${{ matrix.container }}
     strategy:
       matrix:
-        container: ['ubuntu:20.04','ubuntu:22.04']
+        container: ['ubuntu:20.04','ubuntu:22.04', 'ubuntu:24.04']
```
[the remark this answers is not in the document]
Comment (SwooshyCueb) [reply .github/workflows/build-irods-ubuntu.yml]: Fixed

**Review by alanking — APPROVED** [review]
\# it

**Comment by SwooshyCueb** [comment]
\#'d

**Pull request merged**